config/config_new_default.yml

```diff
@@ -0,0 +1,29 @@
+# Simulate = whether or not core is running while running fake data
+core:
+    modules:
+        A:
+            - eps
+            - command_ingest
+            - imu
+        B:
+            - antenna_deployer
+        C:
+            - aprs
+            - iridium
+            - telemetry
+    config_save_interval: 60
+aprs:
+    serial_port: /dev/ttyUSB0
+    telem_timeout: 70
+    message_spacing: 1
+    simulate: False
```

Comment:
Remove all occurrences of `simulate: False` because we aren't using simulation modes anymore. When doing this go through all the modules and remove any `if is_simulate` things